src/Native/System.Native/pal_console.cpp

```diff
@@ -458,7 +458,9 @@ static bool InitializeSignalHandling()
     }
 
     // Finally, register our signal handlers
-    struct sigaction newAction = { .sa_flags = SA_RESTART | SA_SIGINFO };
+    struct sigaction newAction;
```

Comment:
I'd rather we not list and initialize each field to zero individually.  Aren't there better options?